project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/GitHubWorkflowStepCustomizer.java

```diff
@@ -46,14 +46,14 @@ private Job getJob(final GitHubWorkflow workflow, final StepCustomization custom
     // [impl->dsn~customize-build-process.replace-step~0]
     private void applyCustomization(final Job job, final StepCustomization customization) {
         switch (customization.getType()) {
-        case REPLACE:
-            job.replaceStep(customization.getStepId(), customization.getStep());
-            break;
-        case INSERT_AFTER:
-            job.insertStepAfter(customization.getStepId(), customization.getStep());
-            break;
-        default:
-            throw new IllegalStateException("Unknown customization type: " + customization.getType());
+            case REPLACE:
```

Comment:
Is this our official formatter setting? My memory says that the old variant is correct. Could you please check the settings?